src/main/java/net/dv8tion/jda/internal/utils/JDALogger.java

```diff
@@ -34,23 +35,43 @@
 public class JDALogger
 {
     /**
-     * Marks whether or not a SLF4J <code>StaticLoggerBinder</code> was found. If false, JDA will use its fallback logger.
+     * Marks whether or not a SLF4J <code>StaticLoggerBinder</code> (pre 1.8.x) or
+     * <code>SLF4JServiceProvider</code> implementation (1.8.x+) was found. If false, JDA will use its fallback logger.
      * <br>This variable is initialized during static class initialization.
      */
     public static final boolean SLF4J_ENABLED;
     static
     {
-        boolean tmp = false;
+        boolean tmp;
```

Comment:
This was set to false for clarity, I don't see an improvement in removing that.

Reply:
I agree actually, not sure why I removed it.